guides/v2.2/cloud/project/project-conf-files_services-elastic.md

```diff
@@ -14,10 +14,10 @@ functional_areas:
 *   Supports stop words and synonyms
 *   Indexing does not impact customers until the reindex operation completes
 
-We support Elasticsearch versions 1.4, 1.7, 2.4, and 5.2. The default version is 2.4. We support Elasticsearch for all environments starting with {{site.data.var.ece}} 2.1 and later. See [Elasticsearch information]({{ site.baseurl }}/guides/v2.2/config-guide/elasticsearch/es-overview.html).
+We support Elasticsearch versions 1.4, 1.7, 2.4, and 5.2. The recommended version is 5.2.
 
 {:.bs-callout .bs-callout-info}
-Elasticsearch 5.2 is only available for 2.2.3 and later. If you are upgrading to {{site.data.var.ee}} 2.1.3, you must change your configuration as discussed in [the 2.1.3 Release Notes]({{ site.baseurl }}/guides/v2.1/cloud/release-notes/CloudReleaseNotes2.1.3.html#cloud-rn-213-es).
+Elasticsearch 5.2 is only available for 2.2.3 and later. If you are upgrading to {{site.data.var.ee}} 2.1.3, you must change your configuration as discussed in [the 2.1.3 Release Notes]({{ site.baseurl }}/guides/v2.1/cloud/release-notes/CloudReleaseNotes2.1.3.html#cloud-rn-213-es). See [Elasticsearch information]({{ site.baseurl }}/guides/v2.2/config-guide/elasticsearch/es-overview.html).
```

Comment:
This may confuse them as to which link they should click. I think this note needs to stay brief and only about upgrading. That release note is probably really old. Maybe you can find a way to eliminate the link to the old notes and direct them to a better place about changing their configuration. Is there a way?

If not, then remove the link to elasticsearch information.

Reply:
I wondered if we even need this note since Cloud customers need to upgrade to at least 2.1.4 to use ece-tools, and after the migration.